src/Chip/Remove.js

```diff
@@ -2,31 +2,34 @@ import React from 'react'
 import propTypes from '@dhis2/prop-types'
 import { css, resolve } from 'styled-jsx/css'
 
-import { Cancel } from '../icons/Cancel'
+import { CancelOutline } from '../icons/Cancel--outline.js'
 import { colors } from '../theme.js'
 
 const containerStyle = css`
     span {
-        display: block;
-        height: 18px;
-        width: 18px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 20px;
+        width: 20px;
```

Comment:
One question about style rules like this one.
Are we suppose to use `spacers` variables from the theme?
This is also valid for app level style files, I'm trying to use variables from the theme as much as possible but in a few occasions I didn't have the variable I needed so I hardcoded the value.

Reply:
> Are we suppose to use spacers variables from the theme? This is also valid for app level style files, I'm trying to use variables from the theme as much as possible but in a few occasions I didn't have the variable I needed so I hardcoded the value.

That has been my approach as well.

Reply:
Where possible `spacer` should be used, yeah. However, a spacer doesn't need to be used unless it is doing 'spacing'. In this example the icon is `16`, but I've set the w/h to `20` to give a larger click area for the user. The unit of `20` here is not working as a generic spacer, it's specific to this element.

For actual spacing between elements (list items, for example) it's always good to use the right `spacer` variable. Sounds like you guys are using it in the intended way 👍

Reply:
Speaking of using the spacer vars, is there a reason that the pixel values are mentioned in the css var names? To me, using relative names (like spacer-sm, spacer-md, spacer-lg, spacer-xl) seem like a nice way to have variables that you can update without invalidating the names.

Reply:
@ismay please excuse my delayed response.

Thinking back, I think the reason the values are mentioned in the spacer variables was because I could not think of a non-confusing alternative. There are not enough size abbreviations to do `spacer-m`, `spacer-l` etc. I solved this with colors by doing `grey100`, `grey200` etc. I thought it would be very confusing to do `spacer100` that was actually `24px`. Mixing values was not nice.

In the end, I justified the choice by saying that the spacer variables were not intended to change, but rather to help achieve a consistent design across apps on an 8pt grid. If we ever want to change those spacers it will be a mess, but, on balance, it seemed like the quickest way to get the spacers up and running.

Reply:
We did a two-step scale with colors and we could have the same strategy for spacers.

With colors we map e.g. `primary900` to `blue900`: https://github.com/dhis2/ui-core/blob/master/src/theme.js#L84

This allows us to redefine the `primary900` if we need to, without changing the definition of `blue900`. We could could say that `spacer-xl` or `spacer900` is `dp24` or what have you.

Reply:
Yeah that seems practical to me 👍